app/scoring/routes.py

```diff
@@ -78,23 +77,13 @@ def user_scores():
     process_scores.center_scores()
     value_scores = process_scores.get_value_scores()
 
-    session_uuid = uuid.uuid4()
-    value_scores["session-id"] = session_uuid
+    quiz_uuid = uuid.uuid4()
+    value_scores["quiz_uuid"] = quiz_uuid
 
     user_uuid = None
     if current_user:
-        user_uuid = current_user.uuid
+        user_uuid = current_user.user_uuid
     process_scores.persist_scores(user_uuid)
 
-    postal_code = parameter["zipCode"]
```

Comment:
The frontend was supposed to switch over to have postal code be a separate endpoint than questionnaire scoring. Did frontend never switch over as they were supposed to? It doesn’t make a lot of sense to have postal code be associated with the questionnaire. Sean made this other endpoint and it should be functional. What’s the issue with it?

Reply:
This one was a bit confusing to follow because I went back and made changes to CM-743 and then merged them into this one to keep the code cumulative. But I didn't delete the postcode endpoint. I removed any postcode processing from the scores endpoint to complete the switchover to using the new dedicated endpoint.